### DISCUSSION TITLE
Quote attribution in Readme

### DISCUSSION THREAD

**Discussion by Alan-FGR** [skypjack/entt#935] [General]
Hello awesome EnTT community! 😃 

I noticed that you have this quote in the Readme.md:
> `EnTT` has been a dream so far, we haven't found a single bug to date and it's super easy to work with

However, you forgot to attribute that quote:
> `EnTT` has been a dream so far, we haven't found a single bug to date and it's super easy to work with
> *-- Every EnTT User Ever*

😉

**Comment by skypjack** [comment]
Ahahah indeed, you're right!! I'll do it right away!! ❤️🙂
